books/computer-science/bake

```diff
@@ -51,15 +55,19 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :computer_science) do |doc|
 
     # create Chapter Review EOC wrapper
     eoc_wrapper = ChapterReviewContainer.v1(chapter: chapter, metadata_source: metadata)
-
     BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
-    BakeChapterSummary.v1(
+    MoveCustomSectionToEocContainer.v1(
       chapter: chapter,
       metadata_source: metadata,
-      klass: 'section-summary',
+      container_key: 'section-summary',
+      uuid_key: '.section-summary',
+      section_selector: 'section.section-summary',
       append_to: eoc_wrapper
-    )
-
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+      title = EocSectionTitleLinkSnippet.v1(page: section.ancestor(:page))
+      section.prepend(child: title)
+    end
```

Comment:
what's the story with this change?

Reply:
I thought BakeChapterSummary will be deprecated since we can use MoveCustomSectionToEocContainer here instead so I updated it.